application/src/main/java/org/togetherjava/tjbot/Application.java

```diff
@@ -5,36 +5,48 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.togetherjava.tjbot.commands.CommandHandler;
+import org.togetherjava.tjbot.config.Config;
 import org.togetherjava.tjbot.db.Database;
 
 import javax.security.auth.login.LoginException;
+import java.io.IOException;
 import java.nio.file.Path;
 import java.sql.SQLException;
 
-/***
+/**
  * Main class of the application. Use {@link #main(String[])} to start an instance of it.
  */
 public enum Application {
     ;
 
     private static final Logger logger = LoggerFactory.getLogger(Application.class);
+    private static final String DEFAULT_CONFIG_PATH = "config.json";
 
     /**
      * Starts the application.
      *
-     * @param args command line arguments - [the token of the bot, the path to the database]
+     * @param args command line arguments - [the path to the configuration file (optional, by
+     *        default "config.json")]
      */
     public static void main(final String[] args) {
-        if (args.length != 2) {
-            throw new IllegalArgumentException("Expected two arguments but " + args.length
-                    + " arguments were provided. The first argument must be the token of the bot"
-                    + " and the second the path to the database.");
+        if (args.length > 1) {
+            throw new IllegalArgumentException("Expected no or one argument but " + args.length
+                    + " arguments were provided. The first argument is the path to the configuration file. If no argument was provided, '"
+                    + DEFAULT_CONFIG_PATH + "' will be assumed.");
         }
-        String token = args[0];
-        String databasePath = args[1];
 
+        Path configPath = Path.of(args.length == 1 ? args[0] : DEFAULT_CONFIG_PATH);
         try {
-            runBot(token, Path.of(databasePath));
+            Config.load(configPath);
+        } catch (IOException e) {
+            logger.error("Unable to load the configuration file from path '{}'",
+                    configPath.toAbsolutePath(), e);
+            return;
+        }
+
+        try {
+            var config = Config.getInstance();
```

Comment:
I wouldn't use `var` here because the type is not obvious.

Reply:
Imo it is very obvious cause `Config.getInstance()`. I dont care though, I can change it.